### ISSUE TITLE
v-model on radio input should be string

### ISSUE THREAD

**Issue by NateScarlet** [johnsoncodehk/volar#369]
https://github.com/johnsoncodehk/volar/blob/1421e48ca374b14de5c30b5c0d4251bff88d2d4e/packages/vscode-vue-languageservice/src/generators/template.ts#L1358-L1359

code modified from https://v3.vuejs.org/guide/forms.html#radio

```vue
<template>
  <div id="v-model-radiobutton">
    <input type="radio" id="one" value="One" v-model="picked" />
    <label for="one">One</label>
    <br />
    <input type="radio" id="two" value="Two" v-model="picked" />
    <label for="two">Two</label>
    <br />
    <span>Picked: {{ picked }}</span>
  </div>
</template>

<script lang="ts">
import { defineComponent } from 'vue';

export default defineComponent({
  data() {
    return {
      picked: '',
    };
  },
});
</script>
```

npx vue-tsc --noEmit
```
Radio.vue:3:46 - error TS2322: Type 'string' is not assignable to type 'Booleanish'.

3     <input type="radio" id="one" value="One" v-model="picked" />
                                               ~~~~~~~


Radio.vue:6:46 - error TS2322: Type 'string' is not assignable to type 'Booleanish'.

6     <input type="radio" id="two" value="Two" v-model="picked" />
                                               ~~~~~~~



Found 2 errors.
```

**Comment by johnsoncodehk** [comment]
This is upstream problem, see https://github.com/johnsoncodehk/vue-tsc/issues/58.

**Comment by NateScarlet** [comment]
I still don't know how to fix this after read  johnsoncodehk/vue-tsc#58 .

It seems talking about `value`, but this issue is about `v-model`.

**Comment by johnsoncodehk** [comment]
You can use 0.26.10 for now.
